trino-docs.sh

```diff
@@ -42,71 +42,15 @@ find -H "$VERSION" -type f -name '*.html' -print0 | xargs -0 perl -pi -e \
       window._linkedin_data_partner_ids = ['$LINKEDIN_ID'];
     </script>
     <script async src=\"https://snap.licdn.com/li.lms-analytics/insight.min.js\"></script>
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
   </head>
 @;
 s@<style>
 @<link href=\"https://fonts.googleapis.com/css2?family=Alata&family=Open+Sans:ital,wght%400,300;0,400;0,600;1,400&display=swap\" rel=\"stylesheet\">
     <style>
 @;
-s@<header class=\"md-header\" data-md-component=\"header\">
-@<div id=\"announcement\">
-  <div id=\"announcement-content\">
-    <a href=\"https://github.com/trinodb/trino\" target=\"_blank\">
-      Do you ❤️ Trino? Give us a 🌟 on GitHub <i class=\"fab fa-github\"></i>
-    </a>
-  </div>
-</div>
-<header class=\"md-header\" data-md-component=\"header\">
-@;
-s@</header>@</header>@;
 s@<a href=\"[^\"]*\" title=\"Trino $VERSION Documentation\"@<a href=\"/\" title=\"Trino\"@;
 "
 
-cat <<EOT >>$VERSION/_static/trino.css
```

Comment:
All those tweaks will get in the way for updating in the actual docs output and testing live .. and most of them are only for the banner